pkg/bootstrapper/bindata.go

```diff
@@ -1,5 +1,6 @@
-// Code generated for package bootstrapper by go-bindata DO NOT EDIT. (@generated)
-// sources:
+// Code generated by go-bindata. (@generated) DO NOT EDIT.
+
+ //Package bootstrapper generated by go-bindata.// sources:
```

Comment:
In case you're wondering: Yes this is indeed the header generated by go-bindata v3.1.3 (note the space at the beginning and the missing newline before `// sources`). I've PRed a fix upstream and hopefully it'll be pretty again with the next version